src/modules/mavlink/mavlink_receiver.h

```diff
@@ -275,6 +279,10 @@ class MavlinkReceiver : public ModuleParams
 
 	hrt_abstime			_last_utm_global_pos_com{0};
 
+	bool _is_vtol{false};
+
+	vtol_vehicle_status_s _vtol_vehicle_status{};
```

Comment:
You can also subscribe to `vehicle_status` which contains the flags `is_vtol` and `system_type` (multicopter/fixedwing/rover)

Reply:
Sorry I meant `vehicle_status::vehicle_type` which indicates the current MC or FW mode

Reply:
You are right. Seems that this will be more clear solution.